.github/templates/linux_binary_build_workflow.yml.j2

```diff
@@ -46,7 +46,7 @@ env:
   PYTORCH_FINAL_PACKAGE_DIR: /artifacts
   PYTORCH_ROOT: /pytorch
   SHA1: ${{ github.event.pull_request.head.sha || github.sha }}
-  SKIP_ALL_TESTS: 1
+  SKIP_ALL_TESTS: 0
```

Comment:
Hmm, I though that skip all tests is about buidling wheels without tests rather than not running nightly tests
Do you know when this change was made?